### PR TITLE
initialBackoffTime option

### PR DESCRIPTION
**Pull request by roberttod**
- This backoff time will double every time an error is returned

### CODE REVIEW

**Comment by swagatha-christie** [comment]
By analyzing the blame information on this pull request, we identified @davidpadbury to be a potential reviewer

File: lib/queue-that.js [hunk 1]
```diff
@@ -104,7 +105,7 @@ function createQueueThat (options) {
     log.error('Process error, backing off (' + err.message + ')')
     var errorCount = storageAdapter.getErrorCount() + 1
     storageAdapter.setErrorCount(errorCount)
-    storageAdapter.setBackoffTime(now() + INITIAL_BACKOFF_TIME * Math.pow(2, errorCount - 1))
+    storageAdapter.setBackoffTime(now() + options.initialBackoffTime * Math.pow(2, errorCount - 1))
```
Comment (davidpadbury) [inline lib/queue-that.js]: What's `initial` about it? We're doing this every time from a brief look at it?

**Comment by davidpadbury** [comment]
So just making that value configurable? Looks fine for that.

File: lib/queue-that.js (hunk 1 above)
Comment (roberttod) [inline lib/queue-that.js]: Fair, this is a long winded name.

**Pull request merged**